### PR TITLE
Refactored split to utilize quantity

### PR DESCRIPTION
**Pull request by saralandis**
Small PR for split to use quantity in order objects

### CODE REVIEW

File: client/src/components/BillSummaryPage.jsx [hunk 1]
```diff
@@ -60,8 +59,6 @@ class BillSummaryPage extends React.Component {
           Tip Percentage (%):
           <input type="number" id="tipPercentage" name="tipPercentage" min="0" max="1000" value={this.props.tipPercentage} onChange={this.changeTipPercentage}/><br/>
         </form>
-        <button onClick={this.splitEvenly}>Split Evenly</button>
-        <button onClick={this.splitByItem}>Split by Item</button>
```
Comment (milofultz) [inline client/src/components/BillSummaryPage.jsx:64]: Why are we removing these buttons? Don't we still need them?

File: client/src/components/LoggedIn.jsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ class LoggedIn extends React.Component {
           <label htmlFor="zipCode">Zip Code:</label>
           <input type="number" inputMode="numeric" name="zipCode" value={this.state.zipCode} onChange={this.handleChange} />
 
-          <Link to="/find-restaurant" onClick={this.saveHost}>
+          <Link to="/find-restaurant" >
```
Comment (milofultz) [inline client/src/components/LoggedIn.jsx:43]: Not sure why this is removed here. Is this part of the goal of the PR or something else? Just confused

**Review by milofultz — COMMENTED** [review]
Confused on a couple things and want to know what's up before approving

**Review by dworthm — CHANGES_REQUESTED** [review]
I have the same questions as Milo.

File: client/src/components/BillSummaryPage.jsx (hunk 1 above)
Comment (saralandis) [reply client/src/components/BillSummaryPage.jsx:64]: They are doubled. See further down for the buttons.

File: client/src/components/LoggedIn.jsx (hunk 2 above)
Comment (saralandis) [reply client/src/components/LoggedIn.jsx:43]: This has been removed right now bc it is overriding the host object made at login

**Review by milofultz — APPROVED** [review]

**Review by dworthm — APPROVED** [review]

**Pull request merged**